### PR TITLE
Instant lattice Construction

### LINKED ISSUE #6574: Lattice construction time should be instant
Lattice construction time should be instant, like in /tg/. Expansion into space is very tedious with this current delay, and it'll only be worse when we have toxins research.

At the very least, allow multiple lattices to be constructed simultaneously, like with mining.

### PR DESCRIPTION
**Pull request by ThatDan123**
CL: [Improvement] Lattice Construction is instant, and you're allowed to build multiple at the same time

Fixes #6574

### CODE REVIEW

File: UnityProject/Assets/Scripts/Items/PlaceableTile.cs [hunk 1]
```diff
@@ -140,5 +136,9 @@ private class PlaceableTileEntry
 		[Tooltip("The amount of this item required to place tile.")]
 		[SerializeField]
 		public int itemCost = 1;
+
+		[Tooltip("Time it takes to place the tile")]
+		[SerializeField]
+		public float timeToPlace = 1f;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/PlaceableTile.cs:142]: Why public?

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Items/PlaceableTile.cs:142]: image

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Items/PlaceableTile.cs:142]: stuff needs to access it

Comment (corp-0) [reply UnityProject/Assets/Scripts/Items/PlaceableTile.cs:142]: yeah, I didn't see the inner class in my phone. Also was just awakening e-e

**Comment by waffielz** [comment]
thank youuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuu

**Pull request merged**